docs/2-locking-slashing/README.md

```diff
@@ -0,0 +1,15 @@
+# Locking and Slashing
+
+The ERC900 standard doesn’t define what can happen once someone has staked their tokens. Many use cases need to be able to slash users’ tokens 
+depending on the outcome of certain actions. A lot of interesting applications for staking tokens are built on the fact that a part or the 
+totality of one’s tokens could potentially be lost if they were to act in a bad way. For example, in the case of the [Agreement app](https://github.com/aragon/aragon-apps/tree/master/apps/agreement), 
+proposed actions can be challenged and depending on how the challenge is resolved, the user could lose the staked collateral.
+
+Staking app achieves this by adding locks on top of staked balances. A user can designate multiple lock managers, which can be either a contract 
+or an `EOA`, and a maximum allowance for each manager. The managers will then be able to lock up to that allowed amount of tokens, and to unlock 
```

Comment:
Does this behave similarly to token approvals such that once you've created a lock, your allowance goes down? (I don't think so, right?)